management-packages/standalone-cliplugins/metadata.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: data.packaging.carvel.dev/v1alpha1
+kind: PackageMetadata
+metadata:
+  name: standalone-cliplugins.tanzu.vmware.com
+  namespace: standalone-cliplugins
+spec:
+  displayName: "standalone-cliplugins"
+  longDescription: "Standalone CLIPlugin package provides API resources that allows discovery of standalone plugins with Tanzu CLI"
```

Comment:
**question**: "Standalone CLIPlugin package provides … standalone plugins …" feels a little self-referential. Is there some way to rephrase?

Reply:
Rephrased to `Standalone CLIPlugin package provides API resources that allow discovery of default plugins available with Tanzu CLI. These plugins are context agnostic`